src/sentry/runner/commands/configoptions.py

```diff
@@ -203,20 +209,20 @@ def sync(ctx):
     from sentry.utils import metrics
 
     dry_run = bool(ctx.obj["dry_run"])
-    if dry_run:
-        click.echo("!!! Dry-run flag on. No update will be performed.")
```

Comment:
I don't think this should be in a different place.
Also in the current implementation this string is lost as the presenter is not doing anything with the dry run flag.